src/components/dialogs/parameters/security-analysis/security-analysis-parameters.tsx

```diff
@@ -83,8 +104,58 @@ export const SecurityAnalysisParameters = ({ parametersBackend }) => {
         [snackError, updateParameters]
     );
 
+    const formSchema = getLimitReductionsFormSchema(params.limitReductions);
+    const formMethods = useForm({
+        defaultValues: { [LIMIT_REDUCTIONS_FORM]: [] },
+        resolver: yupResolver(formSchema),
+    });
+
+    const toLimitReductions = useCallback(
+        (formLimits: Record<string, any>[]) => {
+            return params.limitReductions.map(
+                (vlLimits: ILimitReductionsByVoltageLevel, indexVl: number) => {
+                    let vlLNewLimits: ILimitReductionsByVoltageLevel = {
+                        ...vlLimits,
+                        permanentLimitReduction: formLimits[indexVl][IST_FORM],
+                    };
+                    vlLimits.temporaryLimitReductions.forEach(
+                        (temporaryLimit, index) => {
+                            vlLNewLimits.temporaryLimitReductions[index] = {
+                                ...temporaryLimit,
+                                reduction:
+                                    formLimits[indexVl][
+                                        LIMIT_DURATION_FORM + index
+                                    ],
+                            };
+                        }
+                    );
+                    return vlLNewLimits;
+                }
+            );
+        },
+        [params]
+    );
+
+    const { handleSubmit, formState } = formMethods;
+
+    const updateLimitReductions = useCallback(
+        (formLimits: Record<string, any>) => {
+            updateParameters({
+                ...params,
+                limitReductions: toLimitReductions(
+                    formLimits[LIMIT_REDUCTIONS_FORM]
+                ),
+            });
+        },
+        [params, updateParameters, toLimitReductions]
+    );
+
+    useEffect(() => {
+        setHaveDirtyFields(!!Object.keys(formState.dirtyFields).length);
```

Comment:
I see this is used everywhere else, but we should consider using isDirty
to fix later, it's better to keep it homogeneous

Reply:
You mean `formMethods.formState.isDirty`